network-api/networkapi/wagtailpages/migrations/0017_product_voting.py

```diff
@@ -10,7 +10,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('wagtailpages', '0015_auto_20201110_1957'),
-        ('wagtailpages', '0016_merge_20201117_1917.py'),
+        ('wagtailpages', '0016_merge_20201117_1917'),
```

Comment:
so... another migration landed in the mean time. Migrations are the best! >_>;;

Latest is 0018_auto_20201126_2017.py so this'll now be 0019

Reply:
[https://github.com/mozilla/foundation.mozilla.org/tree/pni-snippet-conversion/network-api/networkapi/wagtailpages/migrations](0018 isn't in yet). We're still on 0017 in `pni-snippet-converstion.` 

When there's a migration bump I'll fix up the numbering. Thanks for keeping an eye on these!